packages/thirdweb/src/abi/lib/normalizeSignature.ts

```diff
@@ -0,0 +1,65 @@
+export function normalizeSignature(signature: string): string {
```

Comment:
do we need to export this in viem?

Reply:
yep, I believe this is currently not exported by viem or abitype (I'll double check again on my side as well)